src/wrapper/express.js

```diff
@@ -200,6 +202,8 @@ export function toExpress(pipe, opts = {}) {
         requestHeaders.authorization = `Bearer ${AEM_TOKEN}`;
       } else if (AEM_USER && AEM_PASSWORD) {
         requestHeaders.authorization = `Basic ${Buffer.from(`${AEM_USER}:${AEM_PASSWORD}`).toString('base64')}`;
+      } else if (LOGIN_TOKEN) {
+        requestHeaders.cookie = `login-token=${LOGIN_TOKEN}`;
```

Comment:
you can add it to the params directly.

name it `loginToken` as `cookie` would be more than just that.